enterprise/cli/testdata/coder_wsproxy_--help.golden

```diff
@@ -0,0 +1,17 @@
+Usage: coder workspace-proxy
+
+Manage workspace proxies
+
+Aliases: wsproxy
+
+[1mSubcommands[0m
+    create              Create a workspace proxy
+    delete              Delete a workspace proxy
+    edit                Edit a workspace proxy
+    ls                  List all workspace proxies
+    regenerate-token    Regenerate a workspace proxy authentication token. This
+                        will invalidate the existing authentication token.
+    server              Start a workspace proxy server
```

Comment:
Only for now since the command is hidden: https://github.com/coder/coder/blob/3033a468197b7ccfedf3149340740c12836e4fa0/enterprise/cli/workspaceproxy.go#L21-L21

It's still in experimental. I manually generated the wsproxy though because I know some people are trying it out.

Reply:
I'll drop the manual one and let it generate when we pull it out of experimental.